audeer/core/io.py

```diff
@@ -276,45 +276,52 @@ def file_extension(
 
 def list_dir_names(
         path: typing.Union[str, bytes],
+        *,
+        basenames: bool = False,
 ) -> typing.List:
     """List of folder names located inside provided path.
 
     Args:
         path: path to directory
+        basenames: if ``True`` returns basenames of directories
 
     Returns:
         list of paths to directories
 
     Example:
         >>> path = mkdir('path1/path2')
-        >>> dirs = list_dir_names('path1')
-        >>> os.path.basename(dirs[0])
-        'path2'
+        >>> list_dir_names('path1', basenames=True)
+        ['path2']
 
     """
     path = safe_path(path)
     paths = [os.path.join(path, p) for p in os.listdir(path)]
-    return sorted([p for p in paths if os.path.isdir(p)])
+    paths = sorted([p for p in paths if os.path.isdir(p)])
```

Comment:
It doesn't make a difference, but I think I would sort after the if clause.

Reply:
Yes, that's cleaner. I moved to the very end in both functions.